### PR TITLE
CRUD v5 support

### PR DESCRIPTION
**Pull request by promatik**
Can we have it? I'm actually using this package on my boilerplate, so I was not able to upgrade yet.
By the way, I can't see a way to test this with v5, because I can't install it on a v5 demo project.

### CODE REVIEW

**Comment by tabacitu** [comment]
Yup, sure. For future reference, we have an article on our blog that explains how to force-install a package, even if Composer won't let you - https://backpackforlaravel.com/articles/tutorials/how-to-submit-a-non-trivial-PR-to-a-Laravel-package

Present-me would probably word the title a little bit better than past-me 😅

Please install it like that, test it and see if any other changes are needed. I have a feeling _something_ will be needed, since the `lang` directory has moved in L9, from `resources/lang` to `lang`. But other stuff might break too... 🤷‍♂️

**Comment by tabacitu** [comment]
So to install this in my v5 project right now, I'd do:

```
# step 1
git clone git@github.com:Laravel-Backpack/LangFileManager.git vendor/backpack/langfilemanager

# step 2
# add the service provider (config\app.php)

# step 3
# add the autoload to composer.json

# step 4
composer dump-autoload

# step 5
# follow its installation docs
```

Then you'd have it cloned in `vendor/backpack/langfilemanager`, just like it were installed. When you're done making changes, you can just undo all of the above.

**Comment by guleswine** [comment]
How is it going? I'm really looking forward to the dependency update :)

File: src/app/Http/Controllers/LanguageCrudController.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function setupCreateOperation()
         $this->crud->addField([
             'name' => 'flag',
             'label' => trans('backpack::langfilemanager.flag_image'),
-            'type' => 'browse',
+            'type' => backpack_pro() ? 'browse' : 'text',
```
Comment (promatik) [inline src/app/Http/Controllers/LanguageCrudController.php:68]: @tabacitu this was a browse field, that now belongs to backpack pro, so without backpack pro it becomes a text field.

Other than this, everything's works great 👌

File: composer.json [hunk 2]
```diff
@@ -33,10 +33,10 @@
         }
     ],
     "require": {
-        "backpack/crud": "^4.1.0"
+        "backpack/crud": "^5.0"
```
Comment (promatik) [inline composer.json:36]: Just to make sure, in this release we are dropping support for v4 right?

Comment (tabacitu) [reply composer.json:36]: Yes!

**Comment by guleswine** [comment]
Hurrah!

**Comment by tabacitu** [comment]
I know right?! 😅

@guleswine please notice there's a small change - if you're using the FREE version of Backpack, in the Language form the `browse` field will become a `text` field.

**Pull request merged**